web/client/product/pages/Context.jsx

```diff
@@ -111,7 +111,7 @@ class Context extends React.Component {
         return (
             <>
                 <ConnectedContextTheme />
-                <MapViewerCmp {...this.props} onLoaded={this.onLoaded} />
+                <MapViewerCmp {...this.props} onLoaded={this.onLoaded} loaderComponent={false} />
```

Comment:
This can be done to force the loaderComponent, without adding complexity
```suggestion
                <MapViewerCmp {...this.props} onLoaded={this.onLoaded} loaderComponent={null} />
```